example/src/App.tsx

```diff
@@ -19,7 +19,11 @@ function App() {
     isLoading,
     error,
   } = useDocument<TGenericObject>(idReference)
-  const [selectedDate, setSelectedDate] = useState(new Date())
+  const [selectedDate, setSelectedDate] = useState('2023-11-30T14:13:22.407Z')
```

Comment:
Remove datepicker stuff from App.tsx?

Reply:
Yes, absolutely, I just kept it there for your convenience :)